{{cookiecutter.project_dash_slug}}/Dockerfile

```diff
@@ -28,55 +23,23 @@ RUN python -m pip install --upgrade --user pip setuptools
 COPY --chown=user:user setup.py setup.cfg /srv/
 
 # Create a dummy service directory for the dependency installation.
-RUN mkdir -p /srv/{{cookiecutter.project_slug}}
-
-# Install the package with pip.
-RUN pip install /srv[prod]
+RUN mkdir -p /srv/sypy
```

Comment:
Isn't sypy a specific project? Do we want to hard code that here?

Reply:
ops, yeah you right I forgot to change this when trying it out with a demo project.